src/models/configuration.js

```diff
@@ -15,11 +15,19 @@ const CONFIG_FILES = {
   IDS_CACHE: 'ids-cache.json',
 };
 
-function getConfigPath (configFile) {
-  const configDir = (process.platform === 'win32')
+function getConfigDir () {
+  return (process.platform === 'win32')
     ? path.resolve(process.env.APPDATA, 'clever-cloud')
     : xdg.basedir.configPath('clever-cloud');
```

Comment:
Maybe we can just (and not add `getConfigPath`):  
```suggestion
    ? path.resolve(process.env.APPDATA, 'clever-cloud', configFile)
    : path.resolve(xdg.basedir.configPath('clever-cloud'), configFile);
```

Reply:
I don't understand what you suggest @davlgd